platform-includes/debug-symbols/dart-plugin/flutter.mdx

```diff
@@ -0,0 +1,162 @@
+## Installation
+
+In your `pubspec.yaml`, add `sentry_dart_plugin` as a new dev dependency:
+
+```yaml {filename: pubspec.yaml}
+dev_dependencies:
+  sentry_dart_plugin: ^{{@inject packages.version('sentry.dart.plugin', '2.0.0') }}
+```
+
+## Configuration
+
+The Sentry Dart Plugin requires basic configuration in your `pubspec.yaml` file:
+
+<OnboardingOptionButtons
+  options={[
+    {id: 'dsym', checked: true, disabled: true},
+    'source-context',
+    'source-maps'
+  ]}
+/>
+
+```yaml {filename:pubspec.yaml} {"onboardingOptions": {"source-context": "7-8", "source-maps": "9-10"}}
+sentry:
+  project: ___PROJECT_SLUG___
+  org: ___ORG_SLUG___
+  auth_token: ___ORG_AUTH_TOKEN___
+  # Disable automatic upload of debug symbols
+  # upload_debug_symbols: false
+  # Enable source context
+  upload_source_context: true
+  # Enable source maps
+  upload_source_maps: true
+```
+
+<OnboardingOption optionId="dsym">
+The `upload_debug_symbols` option defaults to `true` when not specified.
+</OnboardingOption>
+
+### Alternative Configuration Methods
+
+In addition to configuring the plugin in `pubspec.yaml`, you can use:
+- Environment variables
+- Properties file
+
+For more information, read the [Sentry Dart Plugin README](https://github.com/getsentry/sentry-dart-plugin/tree/main?tab=readme-ov-file#configuration-optional).
+
+## Building Your Application
+
+Before running the plugin, build your Flutter application with one of the following commands. Obuscated is encouraged for production builds, and will make uploading debug symbols necessary to get readable stack traces.
+
+```bash {tabTitle: Obfuscated}
+flutter build apk --obfuscate --split-debug-info=<output-directory>
+flutter build ios --obfuscate --split-debug-info=<output-directory>
+flutter build macos --obfuscate --split-debug-info=<output-directory>
+flutter build windows --obfuscate --split-debug-info=<output-directory>
+flutter build linux --obfuscate --split-debug-info=<output-directory>
+```
+
+```bash {tabTitle: Standard}
+flutter build apk
+flutter build ios
+flutter build macos
+flutter build windows
+flutter build linux
+```
+
+```bash {tabTitle: Flutter Web}
+flutter build web --release --source-maps
+```
+<Alert>
+For Flutter web run `flutter build web --release --source-maps` to generate source maps.
+</Alert>
+
+## Running the Plugin
+
+After building your application, run the plugin to upload debug symbols:
+
+```bash
+flutter pub run sentry_dart_plugin
+```
+
+<Alert>
+
+If you don't obfuscate your build, the plugin won't upload debug symbols but will still configure source context if enabled.
```

Comment:
This is actually not correct, we don't support source context yet for un-obfuscated builds

see https://github.com/getsentry/sentry-dart/issues/1314

Reply:
so we can remove this part imo